diff --git a/config.mk b/config.mk
index 6b17dd5..5fefaa6 100644
--- a/config.mk
+++ b/config.mk
@@ -153,12 +153,8 @@ BASE_SCRIPT=base.js
 CFLAGS := $(CFLAGS)
 CFLAGS += -Wall 
 CFLAGS += -Werror=format-security 
-CFLAGS += -pipe
-CFLAGS += --ansi
 CFLAGS += -std=c99
 CFLAGS += -D_POSIX_C_SOURCE='200112L'
-CFLAGS += -O2
-CFLAGS += -g
 CFLAGS += -D_DEFAULT_SOURCE
 CFLAGS += -D_BSD_SOURCE
 CFLAGS += -D_NETBSD_SOURCE
